package.json

```diff
@@ -55,67 +47,55 @@
+    "@jest/types": "^28.1.3",
     "@swc/jest": "^0.2.21",
-    "@testing-library/jest-dom": "^5.16.3",
-    "@testing-library/react": "^12.1.4",
-    "@testing-library/user-event": "^14.0.0",
-    "@types/jest": "^27.4.1",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^14.2.1",
+    "@types/jest": "^28.1.4",
     "@types/jest-axe": "^3.5.4",
-    "@typescript-eslint/eslint-plugin": "^5.20.0",
-    "@typescript-eslint/parser": "^5.20.0",
-    "babel-jest": "^27.5.1",
-    "core-js": "^3.21.1",
+    "@types/node": "^18.6.1",
+    "@typescript-eslint/eslint-plugin": "^5.31.0",
+    "@typescript-eslint/parser": "^5.31.0",
+    "babel-loader": "^8.2.5",
+    "babel-preset-jest": "^28.1.3",
+    "case-sensitive-paths-webpack-plugin": "^2.4.0",
+    "core-js": "^3.24.0",
     "css-loader": "^6.7.1",
-    "eslint": "^8.11.0",
+    "eslint": "^8.20.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "gh-pages": "^3.2.3",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.5.1",
+    "eslint-config-react-app": "^7.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.6.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.30.1",
+    "espree": "^9.3.2",
+    "history": "^5.3.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^28.1.3",
     "jest-axe": "^6.0.0",
-    "prettier": "^2.6.0",
-    "react-scripts": "5.0.0",
+    "jest-environment-jsdom": "^28.1.2",
+    "jest-resolve": "^28.1.3",
+    "jest-watch-typeahead": "^2.0.0",
+    "prettier": "^2.7.1",
+    "react-app-polyfill": "^3.0.0",
+    "react-refresh": "^0.14.0",
+    "regenerator-runtime": "^0.13.9",
     "style-loader": "^3.3.1",
-    "ts-jest": "^27.1.4",
-    "typescript": "^4.6.3"
-  },
-  "jest": {
-    "globalSetup": "<rootDir>/src/__tests__/utils/globalSetup.ts",
-    "preset": "ts-jest",
-    "setupFilesAfterEnv": [
-      "<rootDir>/src/__tests__/utils/setupTests.ts"
-    ],
-    "testEnvironment": "jsdom",
-    "modulePathIgnorePatterns": [
-      "utils"
-    ],
-    "transform": {
-      "^.+\\.(t|j)sx?$": [
-        "@swc/jest",
-        {
-          "jsc": {
-            "transform": {
-              "react": {
-                "runtime": "automatic"
-              }
-            }
-          }
-        }
-      ],
-      "^.+\\.svg$": "<rootDir>/src/__tests__/utils/fileTransformer.js"
-    },
-    "moduleNameMapper": {
-      "\\.(css)$": "identity-obj-proxy"
-    }
+    "ts-jest": "^28.0.5",
+    "ts-loader": "^9.3.1",
+    "typescript": "^4.7.4",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.9.3",
+    "webpack-merge": "^5.8.0"
```

Comment:
webpack-dev-server is for running the app locally, webpack-merge is used to merge configuration files